test/e2e/move.update.allowed.test.js

```diff
@@ -1,29 +0,0 @@
-import {
```

Comment:
This file has been removed  - is it no longer needed because of the youth risk assessment feature?

Reply:
Yes, the changes mean this isn't a relevant test any more so was cleaning it up.